pages/apim/3.x/kubernetes/apim-kubernetes-operator-architecture.adoc

```diff
@@ -0,0 +1,46 @@
+[[apim-kubernetes-operator-architecture]]
+= GKO Architecture
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_architecture.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+== About Gravitee Kubernetes Operators
+
+A link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes operator^] is a method of packaging, deploying, and managing a Kubernetes application. A Kubernetes application is both deployed on Kubernetes and managed using the Kubernetes API and `kubectl` tooling.
+
+In this context, a Kubernetes operator is an application-specific controller that extends the functionality of the Kubernetes API to create, configure, and manage application instances.
+
+== Architecture overview
+
+The current functionality of the Gravitee Kubernetes Operator (GKO) allows for three main deployment scenarios, as described below.
+
+NOTE: To learn how to deploy GKO on a production cluster, see the link:{{ '/apim/3.x/apim_kubernetes_operator_installation_cluster.html' | relative_url }}[production deployment] section. 
+
+=== Standard deployment
+
+In the standard deployment scenario, the Management API and the API Gateway are deployed in the same Kubernetes cluster.
+
+With this workflow, the GKO listens for link:{{ '/apim/3.x/apim_kubernetes_operator_definitions.html' | relative_url }}[Custom Resource Definitions (CRDs] and for each custom resource, a link:https://kubernetes.io/docs/concepts/configuration/configmap/[ConfigMap^] is created and the API is pushed to the Management API using the import endpoint.
+
+The API Gateway uses the link:https://github.com/gravitee-io/gravitee-kubernetes/tree/master/gravitee-kubernetes-client[Kubernetes Client^] to listen for `ConfigMap`s (with a custom label), and deploys the APIs accordingly.
```

Comment:
(just a personal feeling)
I'm wonder if `ConfigMap` is not an implementation detail and not really usefull for customer architecture. 🤷‍♂️

Reply:
One for @kamiiiel I suppose?:)

Reply:
maybe 😁  
I mean that these schemas are for defining the internal working architecture. but not the architecture of the use from the customer point of view

Reply:
I've removed all configMap mentions to avoid confusion:)